operations/operations/publishing/publishing-guidelines/voice-tone-and-writing-style-guidelines.md

```diff
@@ -1,24 +1,98 @@
 
 * Use simple, plain language that is easy to understand
 * Use direct, casual tone instead of an informal tone
 * Use the active voice
+* Write clearly - avoid vague terms and jargon
+* Write short, active sentences and maintain a visual separation between page elements
 
-**Writing Style**
+The appropriate tone differs across different mediums. We don’t write help documentation with the same tone as website copy.
+You can vary your tone to fit the situation, just as you’d talk to an angry person with a different tone than with an excited child. Voice is to tone as climate is to weather. The Mattermost voice remains the same, even when the tone varies.
 
-* Write clearly and concisely - get to the point quickly and concisely without losing valuable information
-* Work on the premise that "Every page is page 1", as a large portion of users enters our documentation from a Google search
+## Writing Tenets
+
+* Work on the premise that "Every page is page 1", as a large portion of users enters our documentation from a Google search**
+* Add a summary to the top of each page for readers to be able to quickly assess the content for suitability
+* Write clearly and concisely - get to the point quickly without losing valuable information
 * Focus on what the target audience wants to accomplish by being practical/outcome-focused
 * Break long sections into smaller, easily digestable subpages
 * Have at most one key point or action per paragraph
+* Refer to one thing or idea with the same word throughout the page.
 * Use ordered lists or bullets where appropriate, as they are generally easier to read than long blocks of text
 * Write for an international audience without idioms or expressions that people outside of your region are unlikely to understand
+* Minimize content so it can be found and remembered. Keep pages short, modular, and focused on a single topic.
+
+## Writing Style
+
+**Numbers**
+
+Spell out numbers when the number is the first word in a sentence or is less than or equal to ten, otherwise use numeric digits. Use commas to make long numbers easier to read.
+
+**Capitalization**
+
+Use title case for page titles and section titles.
+
+Title case: Grammar, Spelling, and Mechanics
+Sentence case: Language and spelling
+
+## Avoid Fluff
+
+We avoid using the following words and phrases as they're overused and vague:
+
+* Work better
+* Do better work, faster
+* Get more done in less time
+* Chat (as a stand alone feature reference. However it is ok to use chat when describing a benefit "Create a Jira ticket without leaving your chat window" )
+* DevOps teams 
+* Utilize (instead, use “use”)
+* High performance teams
+* Phrases that directly praise ourselves: “We’ve built an intuitive workplace messaging solution”, “It’s a joy to use”
+
+## Writing an MVP
+
+When you submit a PR for your contribution, please include documentation with it. The documentation can be submitted as a new file using the following format:
```

Comment:
Propose writing an example of what a sample "MVP" would look like - or referencing an existing example

Reply:
Good point. 

Next step for this is to highlight it - maybe using a code block. And then putting together an actual example which would probably also ensure there's no confusion as to the content.